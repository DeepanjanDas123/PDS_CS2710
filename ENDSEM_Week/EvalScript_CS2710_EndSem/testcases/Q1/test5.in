BuildTreeFromFrequencyTable
9
N 44
4 1
q 8
p 26
z 41
_ 14
6 18
U 15
l 46
Quit
