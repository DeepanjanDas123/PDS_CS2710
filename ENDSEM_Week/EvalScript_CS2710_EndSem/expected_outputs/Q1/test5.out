((N,l),((((4,q),_),p),((U,6),z)));
